### DISCUSSION TITLE
Multitap PS2 BlueRetro

### DISCUSSION THREAD

**Discussion by souzafl** [darthcloud/BlueRetro#1092] [Ideas]
Friends, a question: I know that it is possible to synchronize 4 joysticks, but only 2 work, as all the diagrams I found only show the connection of the 2 PS2 inputs. Would there be any way to use the other ESP32 GPIOs, and connect them to each input of the Multitap PS2, and for it to use the 4 joysticks for multiplayer games? My PS2 is model 50001, with an esp32 module placed internally.

**Comment by FatBaldDad** [comment]
To me that seems logical.

I would like to mod a multitap with two blue retro cores and install internally in the PS2 console.  You would need to figure out how to enable/disable the multitap for certain games.  Maybe assign a button combo to enable or disable the multitap.  I have been thinking about this.  It would be sweet to see the PS2s function more like a modern console and not be fumbling with connecting/disconnecting devices on these old consoles.

I commented not becuase I know anything but just trying to help.

**Comment by darthcloud** [comment]
Ps1 multitap is supported, but PS2 is not possible because it's too fast for the ESP32 to handle.

**Reply by zeus86** [reply]
This would certainly be worth an own section in the docs. Not the ps2-multitap-question. but "general console limitations (by system)". while basically everyone knows that an NES-Zapper only works on CRTs, the multitap-limitation is something noone would have expected i think. 

...so, if someone makes a 4-player-adapter and plugs this into a multitap (or even better, adds an esp inside a multitap), that should work, right?
